### PR TITLE
Added many German translations.

### PR DESCRIPTION
**Pull request by dlamkins**
Adds many German translations to the module repo, overlay settings, and a few more spots.

### CODE REVIEW

File: Blish HUD/Strings/GameServices/Gw2ApiService.de.resx [hunk 1]
```diff
@@ -153,4 +153,66 @@
...
+  </data>
+  <data name="CreateTokenInstructions_Step4" xml:space="preserve">
+    <value>Kopiere den neuen Schlüssel.</value>
+  </data>
+  <data name="CreateTokenInstructions_Step5" xml:space="preserve">
+    <value>Füge den Schlüssel in das Feld oben ein
+und drücke "Registrieren".</value>
+  </data>
+  <data name="CreateTokenInstructions_Warning" xml:space="preserve">
+    <value>Es kann ein Schlüssel mit wenig Berechtigungen
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Gw2ApiService.de.resx]: wenig -> weniger

File: Blish HUD/Strings/GameServices/Gw2ApiService.de.resx [hunk 2]
```diff
@@ -153,4 +153,66 @@
...
+    <value>Laden von Schlüssel fehlgeschlagen.</value>
+  </data>
+  <data name="TokenStatus_InvalidToken" xml:space="preserve">
+    <value>Ungültiger Schlüssel</value>
+  </data>
+  <data name="TokenStatus_MissingMinPermission" xml:space="preserve">
+    <value>Erforderliche Berechtigungen fehlen</value>
+  </data>
+  <data name="TokenStatus_PartialPermission" xml:space="preserve">
+    <value>Schlüssel fehlt Berechtigungen</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Gw2ApiService.de.resx]: Schlüssel fehlen Berechtigungen

File: Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="PkgRepoSection" xml:space="preserve">
+    <value>Modulbibliothek</value>
+  </data>
+  <data name="PkgRepo_ProviderExtraOption_ReloadRepository" xml:space="preserve">
+    <value>Neuladen der Bibliothek erzwingen</value>
+  </data>
+  <data name="PkgRepo_ProviderExtraOption_FilterSupportedVersions" xml:space="preserve">
+    <value>Zeige Nur Unterstützte Versionen</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx]: Zeige nur unterstützte Versionen

File: Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+  </data>
+  <data name="PkgRepo_ProviderExtraOption_FilterModulesWithUpdates" xml:space="preserve">
+    <value>Zeige Nur Module mit Updates</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx]: Zeige nur Module mit Updates

File: Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  </data>
+  <data name="PkgRepo_ProviderExtraOption_FilterInstalledModules" xml:space="preserve">
+    <value>Zeige Nur Installierte Module</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx]: Zeige nur installierte Module

File: Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+  </data>
+  <data name="PkgRepo_ProviderExtraOption_FilterNotInstalledModules" xml:space="preserve">
+    <value>Zeige Nur Nicht-Installierte Module</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/Modules/RepoAndPkgManagement.de.resx]: Zeige nur nicht installierte Module

File: Blish HUD/Strings/GameServices/OverlayService.de.resx [hunk 7]
```diff
@@ -156,4 +156,22 @@ Guild Wars 2, Heart of Thorns und alle zugehörigen Logos und Designs sind Waren
 eingetragene Warenzeichen der NCSOFT Corporation. Alle anderen Warenzeichen sind das Eigentum
 ihrer jeweiligen Besitzer.</value>
   </data>
+  <data name="Setting_AudioDevice_Description" xml:space="preserve">
+    <value>Dies bestimmt das Audioausgabegerät welches von Blish HUD verwendet wird. Das GW2-Ausgabegerät passt sich an das Audiogerät von GW2 an.</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/OverlayService.de.resx]: Bestimmt das Audioausgabegerät welches von Blish HUD verwendet wird. Das Ausgabegerät passt sich an das gewählte Audiogerät von GW2 an.

File: Blish HUD/Strings/GameServices/OverlayService.de.resx [hunk 8]
```diff
@@ -156,4 +156,22 @@ Guild Wars 2, Heart of Thorns und alle zugehörigen Logos und Designs sind Waren
...
+  </data>
+  <data name="Setting_AudioDevice_DisplayName" xml:space="preserve">
+    <value>Audiogerät</value>
+  </data>
+  <data name="Setting_UseGameVolume_Description" xml:space="preserve">
+    <value>Wenn aktiviert, wird Blish HUD seine Lautstärke an die des Spiels anpassen.</value>
```
Comment (greaka) [inline Blish HUD/Strings/GameServices/OverlayService.de.resx]: This wording declares bhud as masculine. Should it be kept?

**Review by greaka — CHANGES_REQUESTED** [review]

File: Blish HUD/Strings/GameServices/OverlayService.de.resx (hunk 8 above)
Comment (TheMrMilchmann) [reply Blish HUD/Strings/GameServices/OverlayService.de.resx]: This is not entirely true. In this context "seine" is the correct translation for both "his" (masculine) or "its" (neutral).

Comment (dlamkins) [reply Blish HUD/Strings/GameServices/OverlayService.de.resx]: I'm afraid I'm unfamiliar with the significance here as I only speak English.  😕 So I will have to defer on what the appropriate sentence should be.

Comment (greaka) [reply Blish HUD/Strings/GameServices/OverlayService.de.resx]: MrMilchmann is right. It's not clear. An alternative would be `seine` -> `die`. That has the issue that it introduces an article that references bhud's volume which only becomes clear through the second `die` schortly after. It makes the sentence a little more complex. I would probably still use `die` as `seine` diverges from how the rest of the settings descriptions are written.

**Review by greaka — APPROVED** [review]

File: Blish HUD/Strings/GameServices/OverlayService.de.resx (hunk 8 above)
Comment (dlamkins) [reply Blish HUD/Strings/GameServices/OverlayService.de.resx]: Thank you both for the feedback on this.  🙂

**Pull request merged**